### PR TITLE
Fix Ansible-Lint-Errors upcoming after the last Version update

### PR DESCRIPTION
**Pull request by ra-beer**
Partly osism/issues#326

Signed-off-by: Ramona Beermann <rautenberg@osism.tech>

### CODE REVIEW

File: roles/hostname/templates/config.j2 [hunk 1]
```diff
@@ -0,0 +1 @@
+{{ inventory_hostname_short }}{{ '\n' }}
```
Comment (osfrickler) [inline roles/hostname/templates/config.j2:1]: You can drop the `{{ '\n' }}` part, the newline is implicit in the template. This was only needed for the copy/content variant.

File: roles/proxy/tasks/RedHat.yml [hunk 2]
```diff
@@ -14,5 +14,5 @@
     path: /etc/yum.conf
     section: main
     option: proxy
-    value: "{{ proxy_proxies['https']default(proxy_proxies['http']) }}"
+    value: "{{ proxy_proxies['https']|default(proxy_proxies['http']) }}"
```
Comment (osfrickler) [inline roles/proxy/tasks/RedHat.yml:17]: Please add spaces around the `|`, I think that's what the linter's `var-spacing` check would say, too.

Comment (ra-beer) [reply roles/proxy/tasks/RedHat.yml:17]: The linter meant that there was no Pipe before. But yes i agree that it is better to add the spaces too. :)

File: roles/hostname/templates/config.j2 (hunk 1 above)
Comment (ra-beer) [reply roles/hostname/templates/config.j2:1]: You are right i oversee that, thank you.

**Review by osfrickler — APPROVED** [review]

**Pull request merged**